### PR TITLE
Add hack day article

### PR DESCRIPTION
**Pull request by joshua-levin-deliveroo**
An article based on the Hack Day the tech org ran in the summer

### CODE REVIEW

File: _posts/2019-10-02-where-are-they-now.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+From the group: “After identifying several improvement opportunities, we kicked off a working group to address these accessibility issues. We were able to raise greater awareness of accessibility issues, which has been awesome, and have seen a good uptake of colleagues interested in joining the group to evolve it further!”
+
+## Food for the on-call people - Valuable Little Problem
+
+### Daniel
+
+Daniel saw an opportunity to make it easier for on-call engineers to claim their meal allowance by integrating with our PagerDuty schedules.
+
+Daniel: “The food for the on call people project integrated Pagerduty - which is our alerting and notification system for incidents - with our DfB product. When an on call engineer is on call we allow them to claim an allowance to lighten the load in what can be stressful times.
```
Comment (marianeum) [inline _posts/2019-10-02-where-are-they-now.md]: maybe write out Deliveroo for Business?

File: _posts/2019-10-02-where-are-they-now.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+### Erika, Nina, John, Lukas, Florian
+
+<figure>
+![Bot on call team](/images/posts/where-are-they-now/botoncall.png)
+</figure>
+
+The Bot on Call team wanted to use a SlackBot to automate many of the processes involved in our on call processes.
+
+Lukas: “It is definitely still being worked on and we plan to have part of it launched in production this month.”
```
[the remark this answers is not in the document]
Comment (joshua-levin-deliveroo) [reply _posts/2019-10-02-where-are-they-now.md]: There was a lag from when I gathered the quotation and today.  I added a line just below about it being launched.

File: _posts/2019-10-02-where-are-they-now.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title:  "Where are they now?"
+authors:
+  - "Joshua Levin"
+  - "Jenny Sivapalan"
+  - "Troy Harris"
+excerpt: >
+  How Deliveroo does Hack days and then takes it to production
```
[the remark this answers is not in the document]
Comment (joshua-levin-deliveroo) [reply _posts/2019-10-02-where-are-they-now.md]: Fair comment, I'll tweak it.

File: _posts/2019-10-02-where-are-they-now.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+
+<figure>
+![Fruits and pastries](/images/posts/where-are-they-now/food2.png)
+</figure>
+
+<figure>
+![Salads](/images/posts/where-are-they-now/food4.png)
+</figure>
+
+We saw Slackbots to help on-call engineers, how web push notifications could help our riders during the sign up process, and were greatly amused by an IOT solution for monitoring busy toilets.  When the dust settled, winners announced and laptops packed up, it was time to go to the pub next door.
```
[the remark this answers is not in the document]
Comment (joshua-levin-deliveroo) [reply _posts/2019-10-02-where-are-they-now.md]: sure, let me rephrase

**Comment by Crunch09** [comment]
This looks great!
